### PR TITLE
Updated the Assetic commands to the new Symfony style

### PR DESCRIPTION
**Pull request by javiereguiluz**
The output of these commands is very special, so the changes are not really that noticeable:
### assetic:dump

assetic_dump
### assetic:watch

assetic_watch

### CODE REVIEW

File: Command/AbstractCommand.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ private function doDump(AssetInterface $asset, OutputInterface $stdout)
                 }
             }
 
-            $stdout->writeln(sprintf(
+            $stdout->text(sprintf(
```
Comment (stof) [inline Command/AbstractCommand.php]: the typehint of the method should be changed. these methods are not part of ``OutputInterface``

**Comment by stof** [comment]
This will break support for Symfony 2.3, which is not a good idea IMO. If Symfony bundles are dropping support for a supported LTS, it kinds of defeat the purpose of a LTS, and I would like to avoid maintaining multiple branches of the bundle in parallel if possible.

**Comment by MECU** [comment]
http://symfony.com/doc/current/contributing/community/releases.html

Symfony 2.3 is supported until 5/2016 and End Of Life at 5/2017.

**Comment by stof** [comment]
@javiereguiluz now that 2.3 reached its end of maintenance, it may be time to revive this PR to finish it, as dropping support for Symfony 2.3 would now be acceptable.

**Comment by javiereguiluz** [comment]
I'm sorry but I must close this old pull request because now that Assetic is no longer used or recommended in Symfony, I don't want to spend time on this. Others can reuse this PR and finish it though. Thanks!

**Pull request closed without merging**